### PR TITLE
COM-19913: Improve Layout Controller

### PR DESCRIPTION
**Pull request by clash82**
https://jira.ez.no/browse/COM-19913

### CODE REVIEW

**Comment by clash82** [comment]
ping @damianz5 @SylvainGuittard @kamilmusial

File: app/Resources/views/pagelayout.html.twig [hunk 1]
```diff
@@ -55,7 +55,7 @@
             {% block content %}{% endblock %}
 
             {% block footer %}
-                {{  render( controller( "AppBundle:Site:getSettings", { 'template': 'pagelayout_footer.html.twig' } )) }}
+                {{  render( controller( "AppBundle:Layout:getSettings", { 'template': 'pagelayout_footer.html.twig' } )) }}
```
Comment (kamilmusial) [inline app/Resources/views/pagelayout.html.twig]: Maybe ESI here would be a good idea?

Comment (clash82) [inline app/Resources/views/pagelayout.html.twig]: Agree but we'll improve it in separate PR.

**Comment by kamilmusial** [comment]
+1

**Comment by damianz5** [comment]
+1

**Comment by SylvainGuittard** [comment]
+1 but we need an issue ;)

**Pull request merged**